.github/.codecov.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright The Ratify Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+coverage:
+  status:
+    project:
+      default:
+        target: 80%
+        if_ci_failed: error
+    patch:
+      default:
+        target: 80%
```

Comment:
For the core library, I'm considering 90% coverage. What's your opinion? @shizhMSFT

Reply:
We can start with 80%